README.md

```diff
@@ -92,15 +92,26 @@ root@fa76605e3628:/simbricks/experiments# simbricks-run --verbose --force pyexps
 ...
 ```
 
-### Experimental: Interactive SimBricks Jupyter Labs
-
-**This is still a work in progress.**
-
-We are working on a more interactive introduction to SimBricks through Jupyter
-Labs in [this repository](https://github.com/simbricks/labs). These also simply
-require starting a pre-built docker container and then connecting to it from
-your browser. After this you can follow the interactive steps to run SimBricks
-simulation directly from your browser.
+### SimBricks Usage Examples
+
+We provide a [examples-repository](https://github.com/simbricks/simbricks-examples) 
```

Comment:
"an" and "examples-repository" -> "repository" (either way no dash, and examples just shows up on the next line again in the same sentence.